packages/react/src/avatar/Avatar.css.ts

```diff
@@ -44,16 +44,16 @@ export const avatar = recipe({
         yellow: "yellow",
       } as const,
       (color) => ({
-        bg: `${color}.50`,
-        color: `${color}.500`,
+        bg: `${color}.500`,
+        color: "white",
       }),
     ),
     size: {
       xs: { fontSize: "xs", size: "xs" },
-      sm: { fontSize: "sm", size: "sm" },
+      sm: { fontSize: "xs", size: "sm" },
       md: { fontSize: "md", size: "md" },
-      lg: { fontSize: "lg", size: "lg" },
-      xl: { fontSize: "xl", size: "xl" },
+      lg: { fontSize: "2xl", size: "48" },
```

Comment:
let's change the existing `xl` size token from 64 to 48 and use it here
```suggestion
      lg: { fontSize: "2xl", size: "xl" },
```